### PR TITLE
Dev backend advocate bis

### PR DESCRIPTION
**Pull request by manimaarans**

### CODE REVIEW

**Review by atulgupta2024 — CHANGES_REQUESTED** [review]
comments added to the code files. please fix

File: backend/advocate/src/main/java/org/pucar/util/MdmsUtil.java
Comment (subhashini-egov) [inline backend/advocate/src/main/java/org/pucar/util/MdmsUtil.java:1]: How is this service class from the MDMSService?Would it make sense to consolidate?

**Review by subhashini-egov — CHANGES_REQUESTED** [review]
Have suggested changes. Please review @manimaarans

File: backend/advocate/pom.xml [hunk 1]
```diff
@@ -41,21 +41,31 @@
...
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
+            <groupId>org.egov.common</groupId>
+            <artifactId>health-services-common</artifactId>
```
[the remark this answers is not in the document]
Comment (nileshkgbeehyv) [reply backend/advocate/pom.xml:60]: yes

**Review by atulgupta2024 — APPROVED** [review]
there are changes requested in the earlier reviews that are not completed. I am allowing the PR to pass thro' to unblock the team to move forward, but we need to revisit these and fix them later.

**Review by atulgupta2024 — APPROVED** [review]
there are changes requested in the earlier reviews that are not completed. I am allowing the PR to pass thro' to unblock the team to move forward, but we need to revisit these and fix them later.

**Pull request merged**